### PR TITLE
Update uuid.js

### PR DESCRIPTION
**Pull request by sourTaste000**

### CODE REVIEW

**Comment by sourTaste000** [comment]
fixed undefined and null

File: commands/uuid.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const Discord = require("discord.js");
 const fs = require("graceful-fs");
 
 module.exports.run = async (client, message, args) => {
+    if(isNaN(args[0]) || args[0].toLowerCase() === "undefined") return 1;
```
Comment (dominikaaaa) [inline commands/uuid.js:6]: but aren't usernames not gonna be a number almost all the time

Comment (sourTaste000) [reply commands/uuid.js:6]: return 1 stops the module

Comment (sourTaste000) [reply commands/uuid.js:6]: and returns an error

**Comment by sourTaste000** [comment]
nvm im cringe

**Comment by sourTaste000** [comment]
killmelol

**Comment by sourTaste000** [comment]
thats only for c++ lmao

**Comment by sourTaste000** [comment]
why lol

**Comment by humboldt123** [comment]
Its fine as it is.

**Comment by dominikaaaa** [comment]
it isn't tho

it would've completely broke the command

**Pull request closed without merging**